src/lib/mixer/mixer_simple.cpp

```diff
@@ -37,21 +37,6 @@
  * Simple summing mixer.
  */
 
-#include <px4_config.h>
```

Comment:
I am not sure I am up to reviewing the parts of the code where you have removed a large number of includes, however if it still compiles I guess they were not needed